playground/.env

```diff
@@ -6,4 +6,5 @@ HOST_GCP_STAGING=""
 ENABLE_NEW_JSX_TRANSFORM="true"
 ENABLE_EXPERIMENTAL_VITE_BUNDLER="true"
 FAST_REFRESH="true"
-LD_CLIENT_ID_STAGING="5979d95f6040390cd07b5e00"
\ No newline at end of file
+LD_CLIENT_ID_STAGING="5979d95f6040390cd07b5e00"
+LOCAL_HOST="http://localhost:3001"
```

Comment:
Do we need this?

Reply:
Nope. It's not needed anymore. Thanks.

Removed here: 9446cae41fa966890bae8d2f49c790bf0fda335d